operator-common/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlEndpoints.java

```diff
@@ -9,7 +9,9 @@
     STATE("/kafkacruisecontrol/state"),
     REBALANCE("/kafkacruisecontrol/rebalance"),
     STOP("/kafkacruisecontrol/stop_proposal_execution"),
-    USER_TASKS("/kafkacruisecontrol/user_tasks");
+    USER_TASKS("/kafkacruisecontrol/user_tasks"),
+
+    METRICS("/metrics");
```

Comment:
Could you put a comment above this entry and state that this is not provided by Cruise Control and instead comes from the JMX Exporter.

Reply:
Of course :)

Reply:
Tbh I would not add this endpoint here. It's not the code for tests only, it's code related to the Cruise Control integration in the operator and that integration is about the strictly related CC endpoints.